### ISSUE TITLE
Feature request: Webcam Background Removal using deepl learning model "bodyPix"

### ISSUE THREAD

**Issue by v4lentin1879** [v4lentin1879/Giftshot-Animate-GIFs#1]
## Webcam Background Removal
As requested by one of our customers, we would like to enhance the gif editing experience by introducing a background removal feature. Please use a pre-trained deep learning model called `bodyPix` (tensorflow.js) to perform image segmentation of the individual (gif) image frames. The segmentation should label foreground pixels (i.e. the person) and background pixels (i.e. everything else). Subsequently, use the segmentation to replace the real background with a black background on each image frame. 

Your colleague already pre-defined a function `transformSingleGifCanvasFrame(...)` in `~/src/modules/core/screenShot.js` on line 119 to perform the segmentation and image frame manipulation. For this prototype implementation, include external dependencies via html `<script>` tag rather than with `npm install`. You can add the tags in `~/demo/index.html`.

Before getting started, make sure to **create a new branch** and give it **name suitable** for this feature request: `git checkout -b <branch-name>`. 

### While coding
- **Heads up:** `npm run watch` will automatically rebuild the gifshot API upon *save*. However, you'll need to refresh the browser on `localhost:8001` manually to reflect your latest changes.
- The modification is only supposed to work with the *webcam* or *video* GIF source.
